amarachukwu-agbo/random-number-generator
# #165145155 Create endpoint to retrieve numbers

#### What does this PR do?
Create endpoint for retrieving generated phone numbers

#### Description of Task to be completed?
- Add route to get phone numbers
- Add controller to get and sort phone numbers
- Write tests

#### How should this be manually tested?
- Clone the repo and install application dependencies using `yarn`. 
- Run command  `yarn server:start-dev`.
- Using Postman, visit the url `GET http://localhost:3000/api/v1/numbers`, you should see a paginated list of phone numbers.
- Pass in query `sort=ASC` or `sort=DESC` to sort in ascending or descending order
- Pass in query `page=1` or `limit=20` to get a certain page or a certain limit
- Run `yarn test` to test the feature

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145155](https://www.pivotaltracker.com/story/show/165145155)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
